packages/w3up-client/package.json

```diff
@@ -89,6 +89,7 @@
     "assert": "^2.0.0",
     "c8": "^7.13.0",
     "docusaurus-plugin-typedoc": "^0.18.0",
+    "hd-scripts": "^5.0.0",
```

Comment:
medium term I want to remove use of this https://github.com/web3-storage/w3protocol/issues/479

but it does make sense here to be consistent with the other packages ty